main.go

```diff
@@ -92,6 +94,37 @@ func exportArtifacts(artifacts []gradle.Artifact, deployDir string) ([]string, e
 	return paths, nil
 }
 
+func filterUtilityVariants(variants []string) []string {
+	ignoredSuffixes := [...]string{"Classes", "Resources", "UnitTestClasses", "AndroidTestClasses", "AndroidTestResources"}
+	var filteredVariants []string
+
+	for _, v := range variants {
```

Comment:
This block is a perfect example why Go is not a productive language :D

in Kotlin it'd be `return variants.filter{variant -> ignoredSuffixes.none{variant.endsWith(it)}}`